src/Security/RememberLoginHash.php

```diff
@@ -102,7 +102,7 @@ public static function getLogoutAcrossDevices(): bool
         if (is_bool(static::$logoutAcrossDevices)) {
             return static::$logoutAcrossDevices;
         }
-        return static::config()->get('logout_across_devices');
+        return (bool)static::config()->get('logout_across_devices');
```

Comment:
Please don't add code unrelated to the issue being fixed, it's just a distraction that slows downs the pull-request process.

Any sort of attempt to globally enforce type hinting should be handled in an different issue